_field-types/supported-field-types/constant-keyword.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Constant Keyword
+nav_order: 46
```

Comment:
Oh -- I probably need to change the `nav_order`.  This was copied from `keyword.md`.